commercetools-models/src/main/java/io/sphere/sdk/products/PriceDraftDsl.java

```diff
@@ -148,6 +148,10 @@ public PriceDraftDsl withDiscounted(@Nullable final DiscountedPrice discounted)
         return PriceDraftBuilder.of(this).discounted(discounted).build();
     }
 
+    public PriceDraftDsl withTiers(@Nullable final List<PriceTier> tiers) {
+        return PriceDraftBuilder.of(this).tiers(tiers).build();
+    }
+
```

Comment:
I found out the this method was missing too, I think is correct to implement it, what do you think about?
I also added the test to test if it's working properly